src/gleam_stdlib.mjs

```diff
@@ -246,16 +246,28 @@ export function split_once(haystack, needle) {
   }
 }
 
+const unicode_whitespaces = [
+  "\u0020", // Space
+  "\u0009", // Horizontal tab
+  "\u000A", // Line feed
+  "\u000B", // Vertical tab
+  "\u000C", // Form feed
+  "\u000D", // Carriage return
+  "\u0085", // Next line
+  "\u2028", // Line separator
+  "\u2029", // Paragraph separator
+].join();
+
 export function trim(string) {
-  return string.trim();
+  return trim_left(trim_right(string));
 }
 
 export function trim_left(string) {
-  return string.trimLeft();
+  return string.replace(new RegExp(`^([${unicode_whitespaces}]*)`, "g"), "");
 }
 
 export function trim_right(string) {
-  return string.trimRight();
+  return string.replace(new RegExp(`([${unicode_whitespaces}]*)$`, "g"), "");
```

Comment:
It'd probably be a good idea to make this regex a constant so it doesn't get created over and over again.

Reply:
Yes please!

Reply:
Done